src/util/dnd.h

```diff
@@ -109,13 +111,19 @@ class DragAndDropHelper {
                                            const QString& sourceIdentifier,
                                            bool firstOnly,
                                            bool acceptPlaylists) {
+        qDebug() << "dropEventFiles()" << mimeData.hasUrls() << mimeData.urls();
+        qDebug() << "mimeData.hasText()" << mimeData.hasText() << mimeData.text();
+
         if (!mimeData.hasUrls() ||
                 (mimeData.hasText() && mimeData.text() == sourceIdentifier)) {
             return QList<QFileInfo>();
         }
 
         QList<QFileInfo> files = DragAndDropHelper::supportedTracksFromUrls(
                 mimeData.urls(), firstOnly, acceptPlaylists);
+        for (const auto file : files) {
```

Comment:
Please check for Logging::debugEnabled() before executing this loop

Reply:
qDebug() writes always to the mixxx.log file, regardless of the logLevel. 
If we guard the loop, we don't have this info in the log file by default.